remo/profiles/migrations/0057_set_date_left_program_alumni.py

```diff
@@ -0,0 +1,305 @@
+# -*- coding: utf-8 -*-
+import datetime
+from south.db import db
+from south.v2 import DataMigration
+from django.db import models
+
+
+class Migration(DataMigration):
+
+    def forwards(self, orm):
+        """Set the date left program date.
+
+        Set the date to the date of the last report, adding an
+        offset of three months."""
+        from django.core.exceptions import (MultipleObjectsReturned,
+                                            ObjectDoesNotExist)
+
+        alumni_users = orm['auth.User'].objects.filter(groups__name='Alumni')
+
+        for alumni in alumni_users:
+            if alumni.userprofile.date_left_program:
+                continue
+
+            report = None
+            today = datetime.datetime.now()
```

Comment:
nit: you can move this outside the for loop, so all alumnis get the same datetime. It may appear useful in the future.